metrics/metrics.properties

```diff
@@ -0,0 +1,5 @@
+*.sink.jmx.class=org.apache.spark.metrics.sink.JmxSink
```

Comment:
I have a question about this.  Strictly speaking, the default spark metrics that are provided are things like master.aliveWorkers, etc.  Turning on a new sink for those metrics is different than adding a jvm source below.

I'm thinking that our default metrics.properties file should include only the sink, not the jvm source. What do you think?